src/components/ConnectedImagesCarousel.js

```diff
@@ -11,12 +11,9 @@ import { SettingsContext } from '../SettingsProvider';
 import { ImagesCarousel } from './ImagesCarousel';
 import { LoadingContainer } from './LoadingContainer';
 
-export const ConnectedImagesCarousel = ({
-  alternateAspectRatio,
-  navigation,
-  publicJsonFile,
-  refreshTimeKey
-}) => {
+export const ConnectedImagesCarousel = ({ alternateAspectRatio, navigation, publicJsonFile }) => {
+  const refreshTimeKey = `publicJsonFile-${publicJsonFile}`;
+
   const { data, loading, refetch } = useStaticContent({
     refreshTimeKey,
```

Comment:
please do it the same way everywhere: 

```
refreshTimeKey: `publicJsonFile-${publicJsonFile}`,
```

we do not need an extra variable `refreshTimeKey` which is only used once and builds a string.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/80ef948ed83acb0dcc7d7280796271fc8b02d6da